gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -474,18 +624,30 @@ private BlobListOption(StorageRpc.Option option, Object value) {
       super(option, value);
     }
 
+    /**
+     * Returns an option to specify the maximum number of blobs to be returned.
+     */
     public static BlobListOption maxResults(long maxResults) {
       return new BlobListOption(StorageRpc.Option.MAX_RESULTS, maxResults);
     }
 
+    /**
+     * Returns an option to specify the page token from which to start listing blobs.
+     */
     public static BlobListOption startPageToken(String pageToken) {
       return new BlobListOption(StorageRpc.Option.PAGE_TOKEN, pageToken);
     }
 
+    /**
+     * Returns an option to set a prefix for listed blobs names.
```

Comment:
Returns an option to set a prefix to filter results to blobs whose names begin with this prefix.